commons/src/main/java/com/powsybl/commons/reporter/LoggerReporter.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.reporter;
+
+import com.powsybl.commons.PowsyblException;
+import org.apache.commons.text.StringSubstitutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class LoggerReporter extends AbstractReporter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggerReporter.class);
```

Comment:
do we want all these in the same category ?

Reply:
Maybe the "task key" should be used as the logger name ?
Or LoggerReporter.class + task key, so that we can still filter out all log from those "functional" logs in logback config ?

Reply:
PS: this must be mentioned in the implementation's javadoc